lib/account/account-service.js

```diff
@@ -65,9 +58,9 @@ module.exports = function (config, secureHash, Storage) {
      * @param {Object} options
      * @return {Promise.<string>} the directory used to put the file
      */
-    function makeAccountFileAsync(accountInfo, options) {
-        return getDirectoryAsync(accountDirName, accountInfo.accountId).then((accountDir) => {
-            return storage.putAsync(accountDir, JSON.stringify(accountInfo), options).then(() => {
+    function makeAccountFileAsync(accountId, accountFile, options) {
+        return getDirectoryAsync(accountDirName, accountId).then((accountDir) => {
+            return storage.putAsync(accountDir, JSON.stringify(accountFile), options).then(() => {
                 return accountDir;
```

Comment:
Consider - should this return the account file location entirely, not just the directory?

It looks as though `accountDir` is just a bad name for a variable.  If that's the case, rename the variable and update the doc block.